core/src/main/java/com/netease/arctic/io/writer/TaskWriterBuilder.java

```diff
@@ -0,0 +1,17 @@
+package com.netease.arctic.io.writer;
+
+import com.netease.arctic.table.LocationKind;
+import com.netease.arctic.table.OperateKinds;
+import org.apache.iceberg.StructLike;
+import org.apache.iceberg.data.Record;
+import org.apache.iceberg.io.TaskWriter;
+
+public interface TaskWriterBuilder<T> {
```

Comment:
Does table need a `taskWriterBuilfer` method to get the builder?

Reply:
No "taskWriterBuilder" is promised in this case, because the interface is also a "builder"